packages/internals/src/get-generators/getGenerators.ts

```diff
@@ -84,6 +85,8 @@ export async function getGenerators(options: GetGeneratorOptions): Promise<Gener
   if (!fs.existsSync(schemaPath)) {
     throw new Error(`${schemaPath} does not exist`)
   }
+
+  const schemas = (await getSchemaPath(schemaPath))!.schemas
```

Comment:
I think rather than using `!` we can get rid of `existsSync` check above and throw `${schemaPath} does not exist` if it `getSchemaPath` returns `null`.